tests/test_client.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import
+
+from unittest import TestCase
+from unittest import mock
```

Comment:
This only works in Python 3. Is the library going to be Python 3 only? Because in that case you can remove the `from __future__ import absolute_import` parts, those are only for Python 2 :)

Reply:
I personally don't feel the need to support Python 2, since it will be EOL in a year and a half anyway :see_no_evil: What do you think?

Reply:
In general I'm in favor of going Python 3 only, but it depends entirely on which applications this library will be used in. If only HN-API then we should be fine, but if Pino or other Python 2 apps (which we will never get around to upgrading, probably, judging by how the Debian upgrade went) want to use it, then we should support it.

Reply:
Let's go for Python 3. If we ever do decide we want to use it in other systems we will have a good excuse to upgrade other systems to Python 3 too :smirk: